srcpkgs/vlc/template

```diff
@@ -36,7 +38,9 @@ makedepends="
  $(vopt_if v4l2 v4l-utils-devel) $(vopt_if smb samba-devel)
  $(vopt_if lua lua52-devel) $(vopt_if vaapi libva-devel)
  $(vopt_if vdpau libvdpau-devel) $(vopt_if x265 x265-devel)
- $(vopt_if svg librsvg-devel) $(vopt_if sndio sndio-devel)"
+ $(vopt_if svg librsvg-devel) $(vopt_if sndio sndio-devel)
+ $(vopt_if chromecast protobuf-devel) $(vopt_if chromecast protobuf)
```

Comment:
If `protoc` needs to be executed during the build, it should be in `hostmakedepends`, not `makedepends`.

Reply:
Thank you, fixed